dev-tools/composer.json

```diff
@@ -3,6 +3,8 @@
         "php": "^7.1"
     },
     "require-dev": {
+        "humbug/box": "^3.0@alpha",
```

Comment:
is `3.0.0-alpha.0` valid? according to constraint, it is...

Reply:
yes it is, I can add `3.0.0-alpha.5`